rskj-core/src/main/java/co/rsk/peg/utils/RejectedPeginReason.java

```diff
@@ -4,7 +4,9 @@ public enum RejectedPeginReason {
     PEGIN_CAP_SURPASSED(1),
     LEGACY_PEGIN_MULTISIG_SENDER(2),
     LEGACY_PEGIN_UNDETERMINED_SENDER(3),
-    PEGIN_V1_INVALID_PAYLOAD(4);
+    PEGIN_V1_INVALID_PAYLOAD(4),
+    NO_UTXO_OR_UTXO_BELOW_MINIMUM(5),
```

Comment:
Having an `OR` in the name sounds a bit bad to me. It's like it's used for 2 different things (which it is actually). But maybe we could think of a different name

INVALID_AMOUNT?